2996_reflection/reflection.md

```diff
@@ -4163,33 +4181,29 @@ consteval void g(std::meta::info r, X<false> xv) {
+[#]{.pnum} If a `$reflect-expression$` `$R$` matches the form `$qualified-reflection-name$`, it is interpreted as such and represents the entity named by the `$identifier$` of `R` as follows:
 
-[#]{.pnum} A `$reflect-expression$` of the form `^^ $nested-name-specifier$@~_opt_~@ $concept-name$` represents the concept denoted by the `$concept-name$`.
+- [#.#]{.pnum} If the `$identifier$` is a `$namespace-name$` that names a namespace alias ([namespace.alias]), `$R$` represents that namespace alias. For any other `$namespace-name$`, `$R$` represents the denoted namespace.
+- [#.#]{.pnum} Otherwise, if the `$identifier$` is a `$concept-name$` ([temp.concept]), `$R$` represents the denoted concept.
+- [#.#]{.pnum} Otherwise, if the `$identifier$` is a `$template-name$` ([temp.names]) that is an injected-class-name ([class.pre]), the injected-class-name is instead considered a `$type-name$` ([temp.local]) and `$R$` represents the class template specialization so named. For any other `$template-name$` denoting a primary class template, function template, primary variable template, or alias template, `$R$` represents that template.
+- [#.#]{.pnum} Otherwise, the `$qualified-reflection-name$` shall be either a `$type-id$` or an `$id-expression$` (see below) and interpreted as such.
 
-[#]{.pnum} A `$reflect-expression$` of the form `^^ $type-id$` is interpreted as follows:
+[#]{.pnum} A `$reflect-expression$` `$R$` interpreted as `^^ $type-id$` represents an entity determined as follows:
 
-* [#.#]{.pnum} If the `$type-id$` is a `$typedef-name$` that was introduced by the declaration of a template parameter, the `$reflect-expression$` represents the type denoted by that `$typedef-name$`.
-* [#.#]{.pnum} Otherwise, if the `$type-id$` is any other `$typedef-name$`, the `$reflect-expression$` represents the type alias associated with that `$typedef-name$`.
-* [#.#]{.pnum} Otherwise, the `$reflect-expression$` represents the type denoted by the `$type-id$`.
+- [#.#]{.pnum} If the `$type-id$` names a type alias that was not introduced by the declaration of a template parameter, `$R$` represents that type alias.
+- [#.#]{.pnum} Otherwise, `$R$` represents the type denoted by the `$type-id$`.
 
-[#]{.pnum} A `$reflect-expression$` of the form `^^ $id-expression$` is interpreted as follows:
+[#]{.pnum} A `$reflect-expression$` `$R$` interpreted as `^^ $id-expression$` represents an entity determined as follows:
 
-  * [#.#]{.pnum} If the `$id-expression$` denotes an overload set `$S$` and overload resolution for the expression `&$S$` determines a unique function `$F$` ([over.over]{.sref}), the `$reflect-expression$` represents `$F$`.
+  * [#.#]{.pnum} If the `$id-expression$` denotes an overload set `$S$` with no target and overload resolution for the expression `&$S$` determines a unique function ([over.over]{.sref}), `$R$` represents that function.
```

Comment:
What does target mean in this context?

Reply:
In the context of overload resolution for `&S`, it refers to whether there's a "target type" that the selected function should conform to.

Reply:
Doesn't it _never_ have a target type in this context? We're just saying `^^f`.

Reply:
Oops, I think I meant to put that after `&S`.

Reply:
As pointed out by Jens, it needs to go after `&S`.